oat-sa/extension-tao-testqti
# Feature/bsa 208/update German and Dutch translations

Related to: https://oat-sa.atlassian.net/browse/BSA-208

Apply German and Dutch translation.

How to test:
switch to branch
run `php ./tao/scripts/taoUpdate.php`
create a test with a review panel, different tools, timers
create test takers with language German and Dutch
run delivery and check labels
